cloudapi/config.go

```diff
@@ -143,11 +153,16 @@ type Config struct {
 // NewConfig creates a new Config instance with default values for some fields.
 func NewConfig() Config {
 	return Config{
-		Host:                       null.NewString("https://ingest.k6.io", false),
-		LogsTailURL:                null.NewString("wss://cloudlogs.k6.io/api/v1/tail", false),
-		WebAppURL:                  null.NewString("https://app.k6.io", false),
-		MetricPushInterval:         types.NewNullDuration(1*time.Second, false),
-		MetricPushConcurrency:      null.NewInt(1, false),
+		Host:                  null.NewString("https://ingest.k6.io", false),
+		LogsTailURL:           null.NewString("wss://cloudlogs.k6.io/api/v1/tail", false),
+		WebAppURL:             null.NewString("https://app.k6.io", false),
+		MetricPushInterval:    types.NewNullDuration(1*time.Second, false),
+		MetricPushConcurrency: null.NewInt(1, false),
+
+		TracesEnabled:      null.NewBool(false, false),
+		TracesHost:         null.NewString("insights.k6.io", false),
```

Comment:
All the other urls/hosts here include a schema, so at least to be consistent it is likely a good idea to keep it that way.

Reply:
This is a gRPC service. Should I add the `dns:` prefix, then?

gRPC naming: https://github.com/grpc/grpc/blob/master/doc/naming.md

Reply:
Hmmm ... of course grpc does not have `grpc` schema :( 


Does this code support the other ones `ipv4`/`ipv6`  and or `unix` ?